main.go

```diff
@@ -250,8 +252,17 @@ func main() {
 
 	<-consumer.Ready()
 	var minBlock int
+	if cfg.Brokers[0].URL != "null://" {
+		for {
+			time.Sleep(500 * time.Millisecond)
```

Comment:
Put this after the if statement / break so it won't happen if we get the data successfully on the first try.